packages/office-ui-fabric-react/src/components/Slider/Slider.types.ts

```diff
@@ -123,6 +123,24 @@ export interface ISliderProps extends React.ClassAttributes<SliderBase> {
    * @defaultvalue false
    */
   originFromZero?: boolean;
+
+  /**
+
+   * Optional flag to render the thumblabel.
+
+   */
+
+  thumblabel?: boolean;
```

Comment:
`thumblabel` should be camelCased and also the name suggests that this expects a string and not a boolean. Same goes for the `marks` new prop. It doesn’t sound like it would expect a boolean.

Reply:
oh we were thinking of using marks and thumbLabel cause it's the same or at least consistent with the other component libraries. Oh and what did you have in mind as a new name?

Reply:
maybe something along `showThumbTooltip` would give me a hint that this is a boolean prop. Also, a `defaultvalue` documentation field would be nice to have.